### DISCUSSION TITLE
能开发一个记录收藏快照的功能吗

### DISCUSSION THREAD

**Discussion by ZER-ONE** [the1812/Bilibili-Evolved#3512] [功能建议 / Ideas]
我在B站找到的非常好的技术视频，收藏的视频灰了，好难过啊，而又想不起来是哪个视频没了，能开发一个协助收藏的功能吗？就是把收藏的视频的封面保留下来，以及可以给收藏视频做备注，这样我就可以知道这个视频主要讲了什么了。

**Comment by Geeyun-JY3** [comment]
其实有不少脚本和扩展都有找回收藏夹失效视频相关信息的功能，以下是我在用的：

- Btools：https://github.com/imba97/Btools-vue/
- 哔哩哔哩(B站|Bilibili)收藏夹Fix：https://greasyfork.org/scripts/383143

知道 av/BV 号后可以去 BiliPlus 碰碰运气，上述两个脚本/扩展都有直接跳转的方法
